src/main/kotlin/fm/unit/model/Testsuites.kt

```diff
@@ -1,5 +1,24 @@
 package fm.unit.model
 
-data class Testsuite(val report_id: Int, val filename: String)
+/**
+ * A wrapper of the JUnit XML content. It is used to ease the argument mapping for JDBI.
+ */
 data class Payload(val load: String)
-data class TestsuiteSummary(val tests: Int, val errors: Int)
\ No newline at end of file
+
+/**
+ * A testsuite corresponds to a JUnit XMl file. A [[fm.unit.model.Report]] can have multiple testsuites.
+ *
+ * @property filename The JUnit XMl file name
+ * @property payload The string content of the file.
+ */
+data class Testsuite(val filename: String?, val payload: Payload)
```

Comment:
`filename` is `NOT NULL`able in the DB. Why is it nullable here?

Reply:
A filename must not be given in a multipart upload. I changed the SQL for that.